Code.gs

```diff
@@ -99,6 +98,7 @@ function deleteSite(url) {
 function publishContent(summary, type, user, status, url) {
   var doc = DocumentApp.getActiveDocument();
   var title = doc.getName();
+  var folder = DriveApp.getRootFolder();
```

Comment:
@ankitjain28may In reference to your comment https://github.com/dbjpanda/collaborative_editors/pull/12#issuecomment-511682677
I missed adding this line in that Google Docs document
Sorry for such a silly mistake
Please try now